src/plugins/intel_cpu/src/nodes/fullyconnected.h

```diff
@@ -75,6 +79,14 @@ class FullyConnected : public Node {
     static const size_t WEIGHTS_ID = 1;
     static const size_t BIAS_ID = 2;
     dnnl::memory::data_type outputDataType;
+
+    // sparse weights
+    bool useSparseWeights = false;
+    bool printSparseRates = false; // todo: [av] delete
+    int nnzCount = -1;
+    float minSparseRate = 1.f;
+    float weiSparseRate = 0.f;
+    bool canUseSparseWeights();
```

Comment:
I would propose: `useSparseWeightsDecompression` name for better alignment with corresponding configuration key.

Reply:
done